python/cudf_polars/cudf_polars/callback.py

```diff
@@ -185,9 +185,6 @@ def _callback(
     n_rows: int | None,
     should_time: Literal[False],
     *,
-    device: int | None,
-    memory_resource: int | None,
```

Comment:
I think this type signature was incorrect. https://github.com/pola-rs/polars/blame/6259441a9994a4c1f33f03c26a1a1dccd2455fce/py-polars/polars/lazyframe/engine_config.py#L39 claims that `memory_resource` should be a `DeviceMemoryResource | None`. That's what we have on `ConfigOptions` now.